pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -602,11 +576,12 @@ func getDeviceMountPath(volume *reconstructedVolume) (string, error) {
 	}
 }
 
-func (rc *reconciler) updateStates(volumesNeedUpdate map[v1.UniqueVolumeName]*reconstructedVolume) error {
+func (rc *reconciler) updateStates(reconstructedVolumes map[v1.UniqueVolumeName]*reconstructedVolume) error {
 	// Get the node status to retrieve volume device path information.
-	rc.updateDevicePath(volumesNeedUpdate)
+	// TODO: see what volume plugins need this and move to UnmountVolume
```

Comment:
you plan to work on this in the same PR or a different one?

Reply:
I added devicePath update from nodeStatus.